src/models/tests/gamma_poisson_model_test.py

```diff
@@ -218,6 +218,19 @@ def test_overspend(self, mock_gamma_poisson_model):
         self.assertAlmostEqual(gpm.by_impressions([30000]).reach(1), 10000.0, delta=0.1)
         self.assertAlmostEqual(gpm.by_spend([300]).reach(1), 10000.0, delta=0.1)
 
+    def test_fit_histogram_chi2_distance_with_extrapolation(self):
+        # The following point corresponds to
```

Comment:
Perhaps better to say:  The following point is generated under ...

Reply:
I updated this as well.